### PR TITLE
Add fetch manifest script to the feature tests.

### PR DESCRIPTION
**Pull request by smoshiur1237**
This PR will add the fetch manifest script to the feature tests.

### CODE REVIEW

**Comment by kashifest** [comment]
lgtm, run the feature tests also here to check if its working properly

File: Makefile [hunk 1]
```diff
@@ -45,9 +45,11 @@ upgrade_test:
 	make -C ./scripts/feature_tests/upgrade/
 
 feature_tests: setup_env remediation_test cleanup_env pivoting_test
+	./scripts/fetch_manifests.sh
```
Comment (Jaakko-Os) [inline Makefile:48]: would it be more consistent if you define a target,
fetch_manifests:
        ./scripts/fetch_manifests.sh

and then use the 'fetch_manifests' target
feature_tests: fetch_manifests setup_env remediation_test cleanup_env pivoting_test

Comment (smoshiur1237) [reply Makefile:48]: Yes that can be done also.

Comment (smoshiur1237) [reply Makefile:48]: Done

**Comment by fmuyassarov** [comment]
There are no manifests apart from BMH unfortunately.

**Comment by Jaakko-Os** [comment]
looks good!

**Pull request merged**